website/docs/r/sentinel_watchlist_item.html.markdown

```diff
@@ -0,0 +1,113 @@
+---
+subcategory: "Sentinel"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_sentinel_watchlist_item"
+description: |-
+  Manages a Sentinel Watchlist Item.
+---
+
+# azurerm_sentinel_watchlist_item
+
+Manages a Sentinel Watchlist Item.
+
+## Example Usage
+
+### Basic
```

Comment:
we should only have a single example usage for most resources - more complete example usages can be placed in the examples folder?

Reply:
Moved to example folder.